frontend/src/app/providers/auth-service/auth.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from '@angular/core';
+import { AsyncLocalStorage } from 'angular-async-local-storage';
+import { Router } from '@angular/router';
+import { Observable } from 'rxjs';
+import { Http } from '@angular/http';
+
+@Injectable()
+export class AuthService {
+
+  constructor(
+    protected localStorage: AsyncLocalStorage,
+    private router: Router,
+    private http: Http
+  ) { }
+
+  // Sets the logged in user in local storage.
+  setLoggedInUser(credentials: any) {
+    return this.localStorage.setItem('brUser', credentials);
+  }
+
+  login(credentials: any) {
+    this.setLoggedInUser(credentials).subscribe((credentials) => {
```

Comment:
Are we going to make a server request eventually for this? As JWTs will probably only last like a week or two